### PR TITLE
mailing: Product Update 1

### PR DESCRIPTION
**Pull request by spolu**
## Description

Product Update mailing

## Risk

N/A

## Deploy Plan

- deploy `front`

### CODE REVIEW

File: front/mailing/20240308-weekly-update.ts [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    html: `<p>Hi!</p>
+    <p style="max-width: 500px;">
+      Dust makes work work better.
+    </p>
+    <p style="max-width: 500px;">
+      The past month has been packed with powerful new features and upgrades.
+    </p>
+
+    <p style="max-width: 500px; padding-top: 20px;">
+      <b>⛵️ Mistral Large Now Available</b><br/>
```
Comment (fontanierh) [inline front/mailing/20240308-weekly-update.ts:26]: Should we also mention that we can use it for custom assistants ?

File: front/mailing/20240308-weekly-update.ts [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      • Experimental instruction suggestions and website crawling settings.<br/>
+      • New blog posts: <a href="https://blog.dust.tt/pennylane-dust-customer-support-journey/">How Eléonore improved the efficiency of Pennylane’s Care team thanks to Dust</a> and <a href="https://blog.dust.tt/why-dust/">Why Dust</a>.<br/>
+    </p>
+
+    <p style="max-width: 500px; padding-top: 20px;">
+      Happy building,
+    </p>
+    <p>
+      The Dust Team
+    </p>
```
Comment (spolu) [inline front/mailing/20240308-weekly-update.ts:88]: Proposal:

PS: we're still working on getting a proper emailing system. If you want to unsubscribe or have any question about Dust, just reply to the email.

Comment (spolu) [reply front/mailing/20240308-weekly-update.ts:88]: @gabhubert @Yutcam wdyt?

File: front/mailing/20240308-weekly-update.ts [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+}: {
+  user_email: string;
+}) => {
+  const msg = {
+    to: user_email,
+    from: "team@dust.tt",
+    subject: "[Dust] Product Update 1",
+    html: `<p>Hi!</p>
+    <p style="max-width: 500px;">
+      Dust makes work work better.
```
[the remark this answers is not in the document]
Comment (spolu) [reply front/mailing/20240308-weekly-update.ts:19]: @gabhubert @Yutcam wdyt?

**Comment by Yutcam** [comment]
Revised version for Table Queries

📊 **Introducing Table Queries**
• Create custom assistants to query Notion databases, Google Sheets, and CSV.
• Ask questions like "Show me top customers by revenue" and get back instant insights.
• Enables everyone to make data-informed decisions without needing SQL or analytics expertise.

Tables in Dust are automatically kept in sync with their source Google Sheets and Notion databases.

To add at the end of the email
🏗️ **Dust Builders Sessions #1**
Tune in on March 14th for our first Dust Sessions for Builders webinar! Folks from Alan, Pennylane, and Payfit will be demoing their innovative support and sales assistants.

Admin and Builders, save your spot now - you won't want to miss it! Send an email to **[[pauline@dust.tt](mailto:pauline@dust.tt)](mailto:pauline@dust.tt)** to apply.

File: front/mailing/20240308-weekly-update.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (gabhubert) [reply front/mailing/20240308-weekly-update.ts:19]: "with AI assistants" (plural)

**Pull request merged**